recipes/yomm2/all/conanfile.py

```diff
@@ -79,12 +79,7 @@ def generate(self):
         tc.variables["YOMM2_SHARED"] = not bool(self.options.header_only)
         tc.generate()
 
-    def _patch_sources(self):
-        cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
-        replace_in_file(self, cmakelists, "add_subdirectory(docs.in)", "")
-
```

Comment:
This patch still makes sense for the 1.5.1 version, so it'd be better to distinguish between versions. Something like:

```python
        if Version(self.version) <= "1.5.1":
            cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
            replace_in_file(self, cmakelists, "add_subdirectory(docs.in)", "")
```

Where `Version` is imported as `from conan.tools.scm import Version`